boa3_test/test_drive/model/network/payloads/testtransaction.py

```diff
@@ -44,7 +44,7 @@ def to_json(self) -> Dict[str, Any]:
         }
 
     @classmethod
-    def from_json(cls, json: Dict[str, Any]) -> TestTransaction:
+    def from_json(cls, json: Dict[str, Any], *args, **kwargs) -> TestTransaction:
```

Comment:
same question as the `from_json `from boa3_test/test_drive/model/network/payloads/testblock.py